app/templates/home.njk

```diff
@@ -4,65 +4,70 @@
 {% from "govuk/components/breadcrumbs/macro.njk" import govukBreadcrumbs %}
 {% from "govuk/components/input/macro.njk" import govukInput %}
 {% from "govuk/components/inset-text/macro.njk" import govukInsetText %}
-
 {% block pageTitle %}
   {{ siteTitle }}
 {% endblock %}
-
-{% block beforeContent %}
-  
-{% endblock %}
-
+{% block beforeContent %}{% endblock %}
```

Comment:
remove this its an empty block @salfedev

Reply:
done